packages/client/src/containers/RetroPage/RetroPage.css

```diff
@@ -0,0 +1,64 @@
+body {
```

Comment:
I recommend reading a bit about web accessibility and contrast checker because your website is very hard to read.
You can also check this tool that will tell you if the font color is appropriate for the background used.

[Contrast Checker](https://webaim.org/resources/contrastchecker/)